src/styles/theme/color.ts

```diff
@@ -77,17 +89,11 @@ export const oakUiRoles = [
   "bg-decorative5-very-subdued",
   "icon-main",
   "icon-inverted",
-  "border-primary",
-  "border-inverted",
-  "border-neutral",
-  "border-decorative1",
-  "border-decorative2",
-  "border-decorative3",
-  "border-decorative4",
-  "border-decorative5",
-  "border-decorative6",
+  ...oakUiBorderRoles,
```

Comment:
Could be a reasonable idea to restrict this but we haven't done this elsewhere so in the name of consistency lets just stick with uiRoles type . Can you remove this and amend other files accordingly.